lib/sync/util.js

```diff
@@ -7,7 +7,7 @@ var _ = require('underscore');
 var SYNC_LOGGER = 'SYNC';
 var loggers = {};
 
-var generateHash = function (plainText) {
+function generateHash(plainText) {
```

Comment:
interesting to see that you changed from function expression to function declaration. Just curious any reason you doing this?

There is a subtle difference between the 2, and many suggests to use function expressions if you can.